packages/website/zh/docs/development.md

```
@@ -0,0 +1,227 @@
+---
+title: 开发指南
+description: 参与 Principles Disciple 贡献 — 架构、编码规范与工作流。
+---
+
+# 开发指南
+
+想要为 Principles Disciple 贡献代码?本指南涵盖你需要了解的架构、编码规范和开发工作流。
+
+## 项目结构
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)